lib/tasks/csv_to_seed.rake

```diff
@@ -43,13 +42,17 @@ task csv_to_seed: :environment do
     Transaction.create(row.to_hash)
   end
   puts "Loaded transactions.csv"
-
+  
   CSV.foreach('./lib/invoice_items.csv', headers: true) do |row|
     row[4] = row[4].to_i/100.to_f
     InvoiceItem.create(row.to_hash)
   end
   puts "Loaded invoice_items.csv"
-
+  
+  ActiveRecord::Base.connection.tables.each do |t|
+    ActiveRecord::Base.connection.reset_pk_sequence!(t)
+  end
```

Comment:
the primary keys are reset on all tables after the csv files have been used to seed the tables